### PR TITLE
add-looking-for: Add more information about things we want in the test

### PR DESCRIPTION
**Pull request by gadsc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,9 +60,15 @@ SOLUÇÃO
...
 * Uma outra pessoa estará lendo o que você fez, portanto, lembre-se de buscar trazer a semântica do problema no seu código;
-* Crie testes;
 * Não é necessário utilizar nenhum framework, procure utilizar os recursos da linguagem;
 
+GOSTARÍAMOS DE VER
+==================
+* Clean code;
+* Imutabilidade (quando possível);
+* Tratamento de Erro;
+* Separação de clara de responsabilidades (Domínios, Serviços, Repositórios, etc);
```
Comment (vsabreu) [inline README.md]: typo here: `Separação clara de responsabilidades`.

File: README.md [hunk 2]
```diff
@@ -60,9 +60,15 @@ SOLUÇÃO
...
 * Seja criativo;
 * Uma outra pessoa estará lendo o que você fez, portanto, lembre-se de buscar trazer a semântica do problema no seu código;
-* Crie testes;
 * Não é necessário utilizar nenhum framework, procure utilizar os recursos da linguagem;
 
+GOSTARÍAMOS DE VER
+==================
+* Clean code;
+* Imutabilidade (quando possível);
+* Tratamento de Erro;
```
Comment (vsabreu) [inline README.md]: `Tratamento de erros`

**Review by vsabreu — CHANGES_REQUESTED** [review]
small typos

File: README.md [hunk 3]
```diff
@@ -60,9 +60,15 @@ SOLUÇÃO
 * A solução poderá ser desenvolvida em qualquer linguagem da sua escolha;
 * Seja criativo;
 * Uma outra pessoa estará lendo o que você fez, portanto, lembre-se de buscar trazer a semântica do problema no seu código;
-* Crie testes;
 * Não é necessário utilizar nenhum framework, procure utilizar os recursos da linguagem;
 
+GOSTARÍAMOS DE VER
```
Comment (paulinobruno) [inline README.md:65]: Guys, do you think we should state clearer what we want (must have) from what we'd like to see (nice to have)?
I really think *Tests* are not a Nice to Have.

File: README.md [hunk 4]
```diff
@@ -60,9 +60,15 @@ SOLUÇÃO
...
+==================
+* Clean code;
+* Imutabilidade (quando possível);
+* Tratamento de Erro;
+* Separação de clara de responsabilidades (Domínios, Serviços, Repositórios, etc);
+* Testes;
```
Comment (renandsiqueira) [inline README.md]:
```suggestion
* Testes necessários para manter a qualidade do seu código;
```

Comment (renandsiqueira) [inline README.md]:
```suggestion
* Garanta a qualidade do seu código;
```

**Review by otavioucdb — APPROVED** [review]

**Pull request merged**